scripts/orderList.ts

```diff
@@ -0,0 +1,43 @@
+import tokenList from "../optimism.tokenlist.json";
+import fs from 'fs';
+
+async function main() {
+    let updatedTokensList = tokenList;
```

Comment:
nit: not important in this case but in general you would want to [...tokenList] so it's a different object